contrib/pg_tde/documentation/docs/how-to/decrypt-backups.md

```diff
@@ -0,0 +1,38 @@
+# Handling Key Rotation and Encrypted Backups
+
+Backups created with `pg_tde` are encrypted using the *active principal key* at the time of backup. If this key is rotated and the original version is not retained, those backups **can no longer be decrypted**. This makes **secure key storage** essential for reliable disaster recovery.
```

Comment:
The entire `active principal key`, and `current principal key` concept is misleading: a single postgres installation can have multiple active/current principal keys at any time, one for each database. It might have just one, but we support multitenancy which means using multiple keys.